pkg/kapp/cmd/app/deploy.go

```diff
@@ -202,7 +207,7 @@ func (o *DeployOptions) Run() error {
 	if o.DeployFlags.Logs {
 		cancelLogsCh := make(chan struct{})
 		defer func() { close(cancelLogsCh) }()
-		go o.showLogs(supportObjs.CoreClient, supportObjs.IdentifiedResources, existingPodRs, labelSelector, cancelLogsCh)
+		go o.showLogs(supportObjs.CoreClient, supportObjs.IdentifiedResources, existingPodRs, labelSelector, cancelLogsCh, append(meta.LastChange.Namespaces, nsNames...))
```

Comment:
nit: split this into two lines for legibility?

Reply:
Not a fan of splitting things into multiple lines, but if it makes more sense then sure :)

Reply:
No strong feelings about this one, so fine by me